packages/core/config.js

```diff
@@ -90,6 +90,14 @@ module.exports.schema = {
         (accum, method) => accum && typeof value[method] === 'function',
         true
       ))
+  },
+  filters: {
+    defaultValue: () => [ 'password' ],
```

Comment:
should we add any others to this array by default?

Reply:
I think keeping it minimal by default is good, so I'd leave it as this